sdcm/utils/raft/__init__.py

```diff
@@ -68,30 +69,38 @@ class MessageTimeout(NamedTuple):
 ]
 
 
-class RaftFeatureOperations(Protocol):
+class RaftFeatureOperations(ABC):
```

Comment:
question, can you explain why moving between Protocol and ABC ? here, what's the gain ?

Reply:
There are several reasons:
1. IDE show error and warning for some methods inconsistency 
2. Both RaftFeature and NoRaft feature both inherit RaftFeatureOperation class directly, not just implement required methods.

So i decided to switch classic abstract class instead of Protocol to be more clearly